homeassistant/components/braviatv/media_player.py

```diff
@@ -109,24 +109,25 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     }
 
     async_add_entities(
-        [BraviaTVMediaPlayer(client, DEFAULT_NAME, unique_id, device_info)]
+        [BraviaTVMediaPlayer(coordinator, DEFAULT_NAME, unique_id, device_info)]
     )
 
 
 class BraviaTVMediaPlayer(CoordinatorEntity, MediaPlayerEntity):
     """Representation of a Bravia TV Media Player."""
 
     _attr_device_class = DEVICE_CLASS_TV
+    _attr_supported_features = SUPPORT_BRAVIA
 
-    def __init__(self, client, name, unique_id, device_info):
+    def __init__(self, coordinator, name, unique_id, device_info):
         """Initialize the entity."""
 
         self._name = name
-        self._client = client
+        self._coordinator = coordinator
```

Comment:
The coordinator is exposed as `self.coordinator` in the CoordinatorEntity.

Reply:
done